cli/tests/pcluster/aws/test_fsx.py

```diff
@@ -124,3 +125,20 @@ def test_describe_volumes(boto3_stubber):
     # Fourth boto3 call after resetting the AWSApi instance. The latest fsx lifecycle should be retrieved from boto3
     AWSApi.reset()
     assert_that(AWSApi.instance().fsx.describe_volumes([volume])[0]["Lifecycle"]).is_equal_to("CREATED")
+
+
+def get_non_happy_describe_volumes(volume_ids):
+    return MockedBoto3Request(
+        method="describe_volumes",
+        response="Error",
+        generate_error=True,
+        expected_params={"VolumeIds": volume_ids},
+    )
+
+
+def test_non_happy_describe_volumes(boto3_stubber):
```

Comment:
Why do we need this unit tests when we already have the below one `test_non_happy_ontap_and_openzfs_mounting`?

Reply:
This serves as an example of the underlying failure and a test to demonstrate it. However, it could be omitted if it doesn't provide much value beyond `test_non_happy_ontap_and_openzfs_mounting` does.